Source/AcceleratorLattice/AcceleratorLattice.H

```diff
@@ -42,10 +42,15 @@ public:
      * \brief Initialize the element finder instance at the given level of refinement
      *
      * @param[in] lev the level of refinement
+     * @param[in] gamma_boost the Lorentz factor of the boosted frame
      * @param[in] ba the box array at the level of refinement
      * @param[in] dm the distribution map at the level of refinement
      */
-    void InitElementFinder (int lev, amrex::BoxArray const & ba, amrex::DistributionMapping const & dm);
+    void InitElementFinder (
+        int lev,
+        amrex::Real gamma_boost,
```

Comment:
Matching the `const` in AcceleratorLattice.cpp:
```suggestion
        int const lev,
        amrex::Real const gamma_boost,
```